### ISSUE TITLE
Feature Request: Allow setting of USER and TOKEN values instead of forcing the use of os.environ

### ISSUE THREAD

**Issue by daxm** [mapledyne/skytap#10]
I'm a heretic for sure but I'd sure like the ability to set/change the SKYTAP_USER and SKYTAP_TOKEN values after the "import skytap" statement in my Python script.

Right now, I have to import my 'variables.yml' file, parse said file values, set the os.environ values for the 2 values mentioned above and only then can I "import skytap".  Makes the code look ugly.  :-)

As an aside:  I don't have a need for this right now but I *could* see a time when I need to switch from one Skytap user to another user in the middle of a script.  (Think of something like a classroom setup where some VMs are "owned" by the teacher and others are "owned" by a pod user.  I'd need to start up the teacher VMs and then log into each pod user account and start up theirs.)

**Comment by mapledyne** [comment]
It doesn't feel like that'd be a huge change to the scripts to be able to support that. I no longer work with Skytap and don't have a way to test changes (and therefore support this script) any more, but if you wanted to add the changes and check it out, I'd be happy to bring in the PR (or if you're interested in general maintenance of the files, I can set you as a contributor overall)

**Comment by daxm** [comment]
I'll consider it!  Given the age of the repo I'm going to see which is 
less painful.  :-)

**Comment by mapledyne** [comment]
Fair enough. The repo is pretty old - I haven't been able to maintain it in a while so almost certainly there are parts that are outdated at this point. :(
